x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -682,3 +697,34 @@ function sortByName(a: { name: string }, b: { name: string }) {
     return 0;
   }
 }
+
+/**
+ * Return assets for an installed package from ES or from the registry otherwise
+ */
+export async function getPackageAssetsMap({
+  savedObjectsClient,
+  packageInfo,
+  logger,
+}: {
+  savedObjectsClient: SavedObjectsClientContract;
+  packageInfo: PackageInfo;
+  logger: Logger;
+}) {
+  // Retrieve package assets refacto this
```

Comment:
I'm not sure about the comment, leftover?

Reply:
Yes good catch I will cleanup